readline: add getPrompt to get the current prompt (#33675)
nodejs/node: doc/api/readline.md

The `rl.setPrompt()` method sets the prompt that will be written to `output`
whenever `rl.prompt()` is called.

### `rl.getPrompt()`
<!-- YAML
added: REPLACEME
-->

* Returns: {string} the current prompt string

> The `rl.getPrompt()` method gets the prompt that is currently set and returns
> the prompt that will be used for the next `rl.prompt()` call.

Reviewer: ```suggestion
The `rl.getPrompt()` method returns the current prompt used by `rl.prompt()`.
```

The `rl.setPrompt()` method sets the prompt that will be written to `output`
whenever `rl.prompt()` is called.

### `rl.getPrompt()`
<!-- YAML
added: REPLACEME
-->

> * `prompt` {string}

Reviewer 2: ```suggestion
```

The `rl.getPrompt()` method gets the prompt that is currently set and returns
the prompt that will be used for the next `rl.prompt()` call.

> * Returns: {string} the current prompt string

Reviewer 2: This should be moved above the description.
